pkg/coreos/templates/containerd-exec-start

```diff
@@ -0,0 +1,4 @@
+[Service]
+ExecStart=
```

Comment:
What's the reason for this?

Reply:
To override the ExecStart it must be clear first.